manifests/dev/vsphere-7.0u2/guestcluster/1.18/pvcsi.yaml

```diff
@@ -219,6 +245,7 @@ spec:
           args:
             - "--v=4"
             - "--timeout=300s"
+            - "--handle-volume-inuse-error=false"  # Set this to true if used in vSphere 7.0U1
```

Comment:
What if args `handle-volume-inuse-error` not supplied. Does it default to false?

Reply:
No, it defaults to `true`. We need to explicitly set it to `false` to improve online resize efficiency

Reply:
Refer to https://github.com/kubernetes-csi/external-resizer/blob/release-1.0/CHANGELOG-1.0.md#urgent-upgrade-notes for more details